src/app/settings/components/settings-menu/settings-menu.component.scss

```diff
@@ -0,0 +1,43 @@
+.side-menu {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
```

Comment:
I don't know. I barely touched the `settings-menu-component`.
I have now refactored it which removed a lot of the CSS and the markup is now iterating over an array of `MenuItems` instead of having to hardcode every single item